src/helpers/credential.rs

```diff
@@ -264,15 +273,21 @@ impl<'a> Credential<'a> {
         private_key: &str,
     ) -> Result<String, CredentialError> {
         let credential: BbsCredential = serde_json::from_str(credential_str)?;
+        let credential_status = &credential.credential_status.ok_or_else(|| {
+            CredentialError::InvalidCredentialStatus(
+                "Error in parsing credential_status".to_string(),
```

Comment:
```suggestion
                "credentialStatus is required for revocation".to_string(),
```

maybe adjusting the error message, because when you want to revoke, a credentialStatus flag is required